src/components/customizableAside.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CustomizableAvatar from "./components/common/customavatar/CustomizableAvatar"
```

Comment:
I experienced some path-resolving issues with `CustomizableAvatar` and it's CSS file. I corrected these paths locally but wondered if you faced the same issues?

Reply:
@msarit I faced the same thing when trying to implement avatar as well in my single listing page.